src/site/stages/build/process-cms-exports/transformers/paragraph-wysiwyg.js

```diff
@@ -0,0 +1,20 @@
+const { getDrupalValue } = require('./helpers');
+
+function wysiwygTransform(entity) {
+  const { fieldWysiwyg } = entity;
+
+  const transformed = {
+    entity: {
+      entityType: 'paragraph',
+      entityBundle: 'wysiwyg',
+
+      fieldWysiwyg: {
+        processed: getDrupalValue(fieldWysiwyg).replace(/(\r\n|\t)/gm, ''),
```

Comment:
The content here can have carriage returns, newlines, and tabs, but we should be able to ignore those since this is an HTML string.

Reply:
Two things:
1) Was this causing any problems with this extra whitespace?
2) `getDrupalValue` can return non-strings; if it ever does for this field, this will break the build

Reply:
If `getDrupalValue` doesn't return a string for `wysiwyg`, that would mean that our raw schema validation should fail first, right?

---

> 1. Was this causing any problems with this extra whitespace?

Yes - there are discrepancies in the whitespace characters we get from tome-sync and what we currently have with pages.json

### tome-sync sample

![image](https://user-images.githubusercontent.com/2008881/68810850-19395b00-0624-11ea-89f4-74c963a75981.png)


We have `\r\n` after the horizontal rule here as well as `\r\n\r\n` after the `<h2>`

### pages.json sample

![image](https://user-images.githubusercontent.com/2008881/68810792-e8592600-0623-11ea-9222-5b2abaca496c.png)

Here, there isn't a `\r\n` after the horizontal rule, but we do have two `\n`s after the `<h2>`

### Unit testing

The escaped whitespace characters cause the tests to fail:

![image](https://user-images.githubusercontent.com/2008881/68810433-fc505800-0622-11ea-8020-9c68a3daf386.png)